tests/test_run_etl.py

```diff
@@ -0,0 +1,15 @@
+from unittest.mock import patch
+from run_etl import run_etl
+
+
+def test_run_etl():
+    """Test the run_etl function to ensure all steps are called."""
+    with patch("src.setup.Setup.setup_step") as mock_setup, patch(
+        "src.transform.Transform.transform_step"
+    ) as mock_transform, patch("src.load.Load.load_step") as mock_load:
+
+        run_etl()
+
+        mock_setup.assert_called_once()
```

Comment:
You test that each part of the pipeline gets run, but what about results? This is especially important, since you capture exceptions and turn them into print() or `return None`.